docs/0.4/concepts/security_considerations.md

```diff
@@ -0,0 +1,164 @@
+  specify at least one contract registry owner. To add an owner for an existing
+  contract registry, the contract admin uses `scabbard cr update --owners`.
+
+* A namespace owner can change the namespace permissions for the contract. When
+  creating the contract namespace, the contract admin uses `scabbard ns create
+  --owners` to specify at least one contract registry owner. To add an owner for
+  an existing contract namespace, the contract admin uses `scabbard cr update
+  --owners`.
+
+For more information, see [Uploading a Smart
+Contract]({% link docs/0.4/howto/uploading_smart_contract.md %}),
+[the Grid
+README](https://github.com/hyperledger/grid/blob/master/examples/splinter/README.md#demonstrate-grid-smart-contract-functionality),
+and the man pages
+[splinter-circuit-propose(1)]({% link
+docs/0.4/references/cli/splinter-circuit-propose.1.md %}),
+[scabbard-contract-upload(1)]({% link
+docs/0.4/references/cli/scabbard-contract-upload.1.md %}),
+[scabbard-cr(1)]({% link docs/0.4/references/cli/scabbard-cr.1.md %}),
+[scabbard-ns(1)]({% link docs/0.4/references/cli/scabbard-ns.1.md %}),
+and [scabbard-perm(1)]({% link docs/0.4/references/cli/scabbard-perm.1.md %}).
+
+## Smart permissions for organizations and agents with Pike (in Grid)
+
+Pike is a smart contract in Hyperledger Grid that handles organization and
+identity permissions with Sawtooth Sabre. Pike provides organization-specific
+business logic for agents (actors within the organization) and roles (sets of
+permissions).
+
+The predefined admin role in Pike identifies the user or process who can create
+and change agents and roles for that organization. Other roles can be defined by
+organization admins or smart contracts as necessary.
+
+Pike runs as WebAssembly (WASM) code, like other smart contracts. Smart
+permission data is stored in a portion of the Sabre namespace that other smart
+contracts and applications can access, if necessary.
+
+For an example of a smart contract that uses Pike smart permissions, see the
+intkey-multiply smart contract in Hyperledger Sabre.
+
+For more information, see the [Pike Transaction Family
+Specification](https://grid.hyperledger.org/docs/grid/nightly/master/transaction_family_specifications/pike_transaction_family.html),
+[Sawtooth Sabre: Smart
+Permissions](https://sawtooth.hyperledger.org/docs/sabre/nightly/master/smart_permissions.html),
+and [Sawtooth Sabre Application Developer's
+Guide](https://sawtooth.hyperledger.org/docs/sabre/nightly/master/application_developer_guide.html).
+
+## Ports and REST API connections
+
+Splinter uses ports 8084 and 8085 for communication. Other nodes must be able to
+connect on port 8084. (If this is not possible, the Splinter daemon provides a
+work-around for specifying peer nodes explicitly.) Applications and CLIs used to
+manage Splinter will connect to the Splinter REST API on port 8085.
+
+Splinter supports several connection protocols for the `splinterd` REST API
+endpoints:
```

Comment:
The different transport options are for inter-node communication, not the REST API. The REST API is always HTTP.

Reply:
Changed to "for node-to-node communication".